bin/legacy/link.test.js

```diff
@@ -1,5 +1,5 @@
-const { bin } = require('../package.json');
-const { exec } = require('../src/utils');
+const { bin } = require('../../package.json');
```

Comment:
what's `bin/legacy/link.js` about?

Reply:
Adapting commands like `cml-send-comment` to the modern non-hyphenated version, like `cml comment`